blog/2017-09-25-streaming-to-another-database.adoc

```diff
@@ -0,0 +1,222 @@
+= Streaming data to a downstream database
+Jiri Pechanec
+:awestruct-tags: [ mysql, postgres, smt, example ]
+:awestruct-layout: blog-post
+
+Recently there was a short https://www.confluent.io/blog/simplest-useful-kafka-connect-data-pipeline-world-thereabouts-part-1/[series] of blogposts on https://www.confluent.io/[Confluent's] blog.
```

Comment:
How are these posts related to Debezium? As it stands, that sentence is a bit disconnected, esp. given it's the opener.

Reply:
It may make sense to swap the first and second paragraphs so that the first sentence or paragraph says what this post is going to show how to do. The second paragraph can then follow that up by tying it to some of the other recent blog posts. Something like:

> In this blogpost we will try to create a simple streaming data pipeline to continuously capture the changes in a MySQL database and replicate them in near real-time into a PostgreSQL database. We'll show how to do this without writing code by instead using and configuring Kafka Connect, the Debezium MySQL source connector, the Confluent JDBC sink connector, and a few single message transforms (SMTs).
>
> This approach of replicating data through Kafka is really useful on its own, but it becomes even more advantages when we can combine our near real-time streams of data changes with other streams, connectors, and stream processing applications. A recent https://www.confluent.io/blog/simplest-useful-kafka-connect-data-pipeline-world-thereabouts-part-1/[Confluent blog post series] shows a similar streaming data pipeline but using different connectors and SMTs. What's great about Kafka Connect is that you can mix and match connectors to move data between multiple systems.